master/usage/calicoctl/configure/etcd.md

```diff
@@ -5,15 +5,15 @@ canonical_url: 'https://docs.projectcalico.org/v3.1/usage/calicoctl/configure/et
-| `datastoreType`            | `DATASTORE_TYPE`     | Indicates the datastore to use. If unspecified, defaults to `etcdv3`. (optional)      | `kubernetes`, `etcdv3`
-| `etcdEndpoints`            | `ETCD_ENDPOINTS`     | A comma separated list of etcd endpoints. Example: `http://127.0.0.1:2379` (required) | string
-| `etcdUsername`             | `ETCD_USERNAME`      | User name for RBAC. Example: `user` (optional)                                        | string
-| `etcdPassword`             | `ETCD_PASSWORD`      | Password for the given user name. Example: `password` (optional)                      | string
-| `etcdKeyFile`              | `ETCD_KEY_FILE`      | Path to the etcd key file. Example: `/etc/calico/key.pem` (optional)                  | string
-| `etcdCertFile`             | `ETCD_CERT_FILE`     | Path to the etcd client certificate, Example: `/etc/calico/cert.pem` (optional)       | string
-| `etcdCACertFile`           | `ETCD_CA_CERT_FILE`  | Path to the etcd Certificate Authority file. Example: `/etc/calico/ca.pem` (optional) | string
+| Configuration file option | Environment variable | Description                                                                           | Schema
+| --------------------------| -------------------- | ------------------------------------------------------------------------------------- | ------
+| `datastoreType`           | `DATASTORE_TYPE`     | Indicates the datastore to use. If unspecified, defaults to `etcdv3`. (optional)      | `kubernetes`, `etcdv3`
+| `etcdEndpoints`           | `ETCD_ENDPOINTS`     | A comma separated list of etcd endpoints. Example: `http://127.0.0.1:2379` (required) | string
+| `etcdUsername`            | `ETCD_USERNAME`      | User name for RBAC. Example: `user` (optional)                                        | string
+| `etcdPassword`            | `ETCD_PASSWORD`      | Password for the given user name. Example: `password` (optional)                      | string
+| `etcdKeyFile`             | `ETCD_KEY_FILE`      | Path to the file containing the private key of the client certificate issued to `calicoctl` by the etcd certificate authority (CA). Enables `calicoctl` to participate in mutual TLS authentication and identify itself to the etcd server. Example: `/etc/calico/key.pem` (optional) | string
+| `etcdCertFile`            | `ETCD_CERT_FILE`     | Path to the file containing the client certificate issued to `calicoctl` by the etcd CA. Enables `calicoctl` to participate in mutual TLS authentication and identify itself to the etcd server. Example: `/etc/calico/cert.pem` (optional) | string
+| `etcdCACertFile`          | `ETCD_CA_CERT_FILE`  | If the etcd CA is not in the list of trusted root CAs on the `calicoctl` host, use this parameter to supply `calicoctl` with the path to the file containing the root certificate signed by the etcd CA. Configures `calicoctl` to trust certificates signed by the etcd CA. Example: `/etc/calico/ca.pem` (optional) | string
```

Comment:
"containing the root certificate signed by the etcd CA" should be "containing the root certificate for the CA that issued the etcd server certificate"